OpenRA.Mods.Common/Widgets/Logic/TabCompletionLogic.cs

```diff
@@ -19,58 +19,83 @@ public class TabCompletionLogic
 		IList<string> candidates = new List<string>();
 		int currentCandidateIndex = 0;
 		string lastCompleted;
+		public readonly string PrefixOnStart;
+		public readonly string SuffixOnStart;
 		string prefix;
 		string suffix;
 
-		public IList<string> Commands { get; set; }
+		private int longestElementLength;
+		private IList<string> universe;
```

Comment:
Our style guide says the default accessibility modifier should be omitted. i.e. You should remove the explicit `private` here.